src/IndexAndKeyProcessor.ts

```diff
@@ -83,8 +84,18 @@ export default async (conversion: Conversion, tableName: string): Promise<void>
         } else {
             // "schema_idxname_{integer}_idx" - is NOT a mistake.
             const columnName: string = objPgIndices[index].column_name[0].slice(1, -1) + cnt++;
+            let index_name = `${ conversion._schema }_${ tableName }_${ columnName }_idx`;
```

Comment:
@simondh 
Following is not critically important, but:
1. Could you camel-case variables, as all variables in Nmig are camel-cased? (<code> index_name</code> and <code>uuid_tag</code>)
2. Could you remove <code>console.log</code>, please? (line 93)